R/annotateProteinInfoFromIndra.R

```diff
@@ -0,0 +1,115 @@
+#' Annotate Protein Information from INDRA
+#'
+#' This function annotates a given dataframe with additional protein information 
+#' using various APIs. The input dataframe must contain columns 'Protein' and 'IDType'.
```

Comment:
So ID type would be a separate parameter of this function and not in the dataframe itself.

Reply:
Intuition is that datasets will generally have one or the other, but not both.  In the future, if it does have both, then this column would be useful, but I think making it a separate parameter is more user friendly at the moment.

Reply:
done